infra/gcp/ensure-test-storage.sh

```diff
@@ -0,0 +1,27 @@
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# The service account name for the image promoter test account
+PROMOTER_TEST_SVC="k8s-infra-gcr-promoter-test"
```

Comment:
We need to switch the manifests to use this new account too right?

Reply:
Yes but only for the e2e tests inside the promoter repo here: https://github.com/kubernetes-sigs/k8s-container-image-promoter/blob/master/test/tests.yaml#L11, because only the e2e tests will use this svc account.

Reply:
Tracking this with this issue inside the promoter repo: https://github.com/kubernetes-sigs/k8s-container-image-promoter/issues/72